src/models/products/reserves.jl

```diff
@@ -54,19 +59,23 @@ requirement - the required quantity of the product
 """
 struct StaticReserve <: Reserve
     name::String
-    contributingdevices::Array{Device}
+    contributingdevices::Vector{Device}
     timeframe::Float64
     requirement::Float64
+    internal::PowerSystemInternal
+end
 
-    function StaticReserve(name::String,
-                                contributingdevices::Array{Q},
-                                timeframe::Float64,
-                                generators::Array{G}) where {Q <: Device, G <: TechThermal}
+function StaticReserve(
+                       name,
+                       contributingdevices,
+                       timeframe,
+                       generators,
+                      ) where {Q <: Device, G <: TechThermal}
```

Comment:
If the types from the function argument are removed, shouldn't the `where {Q<:Device, G<:TechThermal}` also be removed?

Reply:
We should open an issue to check that all the functions are typed

Reply:
Fixed.  If someone thinks that these should actually be typed, speak up.